dias/config_loader.py

```diff
@@ -207,13 +242,55 @@ def _import_analyzer_class(self, name):
 
     # Data model
     def __getitem__(self, key):
+        """
+        Read a variable from the dias config.
+
+        Parameters
+        ----------
+        key : str
+            Name of the config variable
+
+        Returns
+        -------
+        The value of the config variable.
+        """
         return self.global_config[key]
 
     def __setitem__(self, key, value):
+        """
+        Set or create a variable in the global dias config.
+
+        Parameters
+        ----------
+        key : str
+            The name of the config variable.
+        value
+            The value to set.
+        """
         self.global_config[key] = value
 
     def __contains__(self, key):
+        """
+        Check if the global dias config contains a variable.
+
+        Parameters
+        ----------
+        key : str
+            The name of the variable.
+
+        Returns
+        -------
+        bool
+            `True` if the config contains the variable, otherwise `False`.
+        """
```

Comment:
```
Implements `key in self` and `key not in self`.
```

Reply:
I used 
`Test key in self and key not in self.`